### PR TITLE
Feature/multi timeframe support

### LINKED ISSUE #47: Add multi timeframe support
Create a way to be able to use multiple timeframes for every pair.

Things to be done:
- Make it easy configurable
- Adjust downloading / checking for multiple timeframes
- Make data easy accessible in generate_indicators methods

### PR DESCRIPTION
**Pull request by noudcorten**
# Fixes

Fixes #47 

# What has been changed?

* Dataframe index is now formatted in datetime
* Added function to strategy that can change the timeframe of the current dataframe to a new desired timeframe

# Examples

N/A

# Actions to be performed before this can be merged

N/A

# Security Measures

* Fixed bug where the dataframe was not correctly sorted based on dataframe index.

# Potential Issues/Future considerations

# Additional Info

* Multi timeframe support is achieved by resampling current dataframe.

### CODE REVIEW

File: data/datamodule.py [hunk 1]
```diff
@@ -278,7 +279,7 @@ def check_backtesting_period(self, pair: str, df: DataFrame, final_timestamp: in
         :rtype: DataFrame
         """
         # Get backtesting period
-        index_list = list(df.index.values)
+        index_list = [pd.to_datetime(time).timestamp() * 1000 for time in list(df.index.values)]
```
Comment (dema-trading-ai) [inline data/datamodule.py:282]: Clean.

**Review by dema-trading-ai — APPROVED** [review]
I like the simplicity. I thought we had to download data again for separate timeframes.

File: .gitignore [hunk 2]
```diff
@@ -137,4 +137,3 @@ dmypy.json
 .idea/
 
 **/backtesting-data/**
-.vscode
```
Comment (keebrev) [inline .gitignore:140]: Y is this gone?

Comment (noudcorten) [reply .gitignore:140]: Because it is already ignored on line 132

**Review by keebrev — APPROVED** [review]

**Pull request merged**